rmf_site_editor/src/interaction/camera_controls/mod.rs

```diff
@@ -257,11 +271,12 @@ impl FromWorld for CameraControls {
             .spawn(DirectionalLightBundle {
                 directional_light: DirectionalLight {
                     shadows_enabled: false,
-                    illuminance: 20000.,
+                    illuminance: 50.,
```

Comment:
This is part of a large refactoring that they did for lights.
TLDR I adopted the lights to be more physically accurate, for example the new illuminance value is corresponding to a "family living room lights", compared to the previous that was "full daylight" [ref](https://docs.rs/bevy/0.13.2/bevy/pbr/struct.DirectionalLight.html).
In full daylight, a small lightbulb would have been pretty much invisible, with this setting we can more accurately model indoor environments.

As a result I had to increase the camera exposure to make sure that the scene wouldn't result too dim. The camera exposure value itself is hand-tuned to make the appearance after the migration "close" to the one before.